### DISCUSSION TITLE
Unable to write to Folder Settings (4.19.0)

### DISCUSSION THREAD

**Discussion by jefftuschool** [coder/code-server#6542] [General]
Just update to a version 4.19.0, and got a error message like this. Any ideas?

Unable to write to Folder Settings because workbench.editor.showTabs does not support the folder resource scope.

<img width="1137" alt="Screenshot 2023-11-19 at 9 47 47 AM" src="https://github.com/coder/code-server/assets/107432691/e45a24b3-e312-4d4c-8954-bf45b9844233">

**Comment by code-asher** [comment]
Looks like this was just fixed upstream so hopefully it makes it into the next release, whenever that is: https://github.com/microsoft/vscode/issues/198056
